mods/default/init.lua

```diff
@@ -6,6 +6,7 @@
 -- Definitions made by this mod that other mods can use too
 default = {}
 
+default.path = minetest.get_modpath("default")
```

Comment:
You could make the path local. Other mods can still use `minetest.get_modpath("default")` if they need it, so there is no reason to make it globally available. :)